tests/src/Context/JoinupCommunityContentContext.php

```diff
@@ -260,4 +263,124 @@ public function pinCommunityContentInGroup(AfterNodeCreateScope $scope) {
     }
   }
 
+  /**
+   * Asserts that a sequence of HTML markup exists inside a comment.
+   *
+   * The comment is identified by its '1 based' number. Numbers are starting
+   * from 1 and are assigned in the order comments are shown on the page,
+   * regardless of their indent.
+   *
+   * @param string $comment_number
+   *   The '1 based' number of the comment in the page.
+   * @param string $markup
+   *   The piece of markup to be checked.
+   *
+   * @Then (the )comment #:comment_number should contain the markup :markup
+   */
+  public function assertCommentContainsMarkup(string $comment_number, string $markup): void {
```

Comment:
It is actually possible to type hint the `$comment_number` parameter to the correct type `int`!
Behat has a nice feature that performs type inspection and will cast automatically to the requested type if possible. :+1: 

This can also be done for all other step definitions that are added in this PR that work with integer values.